types/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export default chalk;
```

Comment:
This is currently incorrect because we aren't yet exporting the `default` property. If we agree upon doing that (not my call to make), I'd suggest switching to an interface.

```
export default interface Chalk {
    (...text: string[]): string;
    (text: TemplateStringsArray, ...placeholders: string[]): string;

    enabled: boolean;
    level: Level;
    readonly supportsColor: boolean;

    hex(color: string): Chalk;
    // ...
}
```

Gives much better results in VS Code and looks way cleaner as well.